test/hotspot/jtreg/compiler/unsafe/UnsafeCopyMemory.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @key stress randomness
  * @library /test/lib
+ * @bug 8283122
```

Comment:
We usually don't link test bugs but the `@bug` label should be used to link (VM) bugs that have been found by that test.